release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -196,6 +196,17 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-operators"]
+=== Operators
+
+[id="ocp-4-6-metering-operator"]
+==== Support for retention period for metering Reports
```

Comment:
I think there may be a missing word here as this reads a bit strange to me.

"Support for configuring a retention period of metering Reports" may be a bit wordy.

Reply:
Looking at the 4.5 release notes, we're not entirely consistent with how we write the titles for new support/features. Here's an example from 4.5 for Cluster logging - https://docs.openshift.com/container-platform/4.5/release_notes/ocp-4-5-release-notes.html#ocp-4-5-log-retention-policy

> **New Elasticsearch log retention feature**

What about, "Configuring a retention period of metering Reports" ?

I can get some guidance on wording during the peer review as well.

Reply:
Yeah, that's fine with me :+1: